index.js

```diff
@@ -0,0 +1,149 @@
+const compareLangKeys = () => {
+  // lib is needed to show deuplicated keys
+  const jsonValidator = require('json-dup-key-validator');
+
+  //requiring path and fs modules
+  const path = require('path');
+  const fs = require('fs');
+  const directoryPath = path.join(__dirname, 'locales');
+
+  const mapAllFiles = (files, filesContent) => {
+    const isJsonFile = (JSONdata, file) => {
+      const singleFilesContent = [];
+
+      for (const key in JSONdata) {
+        singleFilesContent.push(key);
+      }
+      filesContent.push({
+        fileName: file,
+        fileKeys: singleFilesContent
+      });
+    };
+
+    const isNotJsonFile = () => {
+      notJSON++;
+      filesContent.push({
+        fileName: 'null',
+        fileKeys: []
+      });
+    };
+
+    const loadFiles = (file, isJsonFile, isNotJsonFile) => {
+      // load file if it is a JSON
+      if (file.includes('.json')) {
+        const JSONdata = require(`./locales/${file}`);
+
+        const isDuplicated = new Promise((resolve, reject) => {
+          fs.readFile(`./locales/${file}`, 'utf8', (err, data) => {
+            if (err) {
+              reject(err);
+            } else {
+              const validateDuplicates = jsonValidator.validate(data, false);
+              if (validateDuplicates) {
+                resolve(
+                  `[DUPLICATED] ${jsonValidator.validate(
```

Comment:
Consider creating some kind of representation to avoid immediate value display. Then you can separate the display logic from duplicate detection. Consider keeping some collection of objects like this: `{ type: Duplicate, erro, file }`. Then after you finish detection, you can display them in a single place.